src/main/java/AddAndNotifyCommand.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class AddAndNotifyCommand implements Command {
+    private final UsersBook usersBook;
+    private final String info = "If you want to add a new user and notify him, enter the command => 'addAndNotify'";
+
+    public AddAndNotifyCommand(UsersBook usersBook) {
+        this.usersBook = usersBook;
+    }
+
+    @Override
+    public void execute() {
+        AddCommand add = new AddCommand(usersBook);
+        add.execute();
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your email to send a notification");
+        String emailTo = scanner.nextLine();
+
+        NotificationData notificationData = new NotificationData();
+        notificationData.setEmailTo(emailTo);
+        notificationData.setEmailNotificationText(
+                "You have successfully registered in our service 'Users book'. " +
+                        "Your ID: " + usersBook.getLastAddedUser().getId());
+
+        NotificationService emailNotification = new EmailNotification();
+        emailNotification.sendNotification(notificationData);
+
+        System.out.println("EMail Sent Successfully!!!");
```

Comment:
Why message about email here ? you  have abstraction NotificationData and NotificationService supposing that might be different implementations, but hardcode information about email